techniques/general/G101.html

```diff
@@ -2,179 +2,153 @@
+               <p>「ドライバー」という用語は、プリンターでは特定の指示をするソフトウェアとして定義されている: 「あなたのプリンタのドライバーをアップデートする必要があるかもしれない。」
                </p>
             </section>
             <section class="example" id="example-4-an-idiomatic-expression">
-               <h3>Example 4: An idiomatic expression</h3>
-               <p>Some people say "spill the beans" when they mean "reveal a secret", e.g., "In the
-                  police station, Joe spilled the beans about the plot to kidnap the prime minister."
+               <h3>事例 4: 固有表現</h3>
+               <p>例えば、「警察署で、ジョーは首相を誘拐するたくらみについて口をすべらした。」のように、「秘密を漏らす」の意味で「口をすべらす」と言う人がいる。
                </p>
             </section>
             <section class="example" id="example-5-an-idiomatic-expression-in-japanese">
-               <h3>Example 5: An idiomatic expression in Japanese</h3>
-               <p>This example uses parentheses to provide the definition of an idiomatic expression
-                  in Japanese. The phrase in Japanese says that "he throws a spoon." It means that there
-                  was nothing he can do and finally he gives up.
+               <h3>事例 5: 日本語の慣用表現</h3>
+               <p>この事例は、日本語の慣用表現の定義を提供するために括弧を使用する。日本語の語句では、「さじを投げる。」と書かれている。これは、彼のできることが何もなく、最終的にあきらめたことを意味している。
                </p>
                <p>
-                  <span lang="jp">さじを投げる（どうすることもできなくなり、あきらめること）。</span>
+                  <span lang="jp">さじを投げる (どうすることもできなくなり、あきらめること)。</span>
                   
                </p>
             </section>
             <section class="example" id="example-6-an-unfamiliar-adopted-foreign-word-in-english">
-               <h3>Example 6: An unfamiliar adopted foreign word in English</h3>
-               <p>Users may not understand the meaning of an unfamiliar word adopted from another language:
-                  "We need to leave town pronto (quickly).
+               <h3>事例 6: 英語で馴染みのない外来語</h3>
+               <p>利用者は、他の言語からの馴染みのない外来語を理解できないかもしれない: 「我々はすぐに (すばやく) 街を離れる必要がある。」
                </p>
             </section>
             <section class="example" id="example-7-unfamiliar-adopted-words-in-japanese">
-               <h3>Example 7: Unfamiliar adopted words in Japanese</h3>
-               <p>In Japanese, Kata-kana is used for adopted foreign words. If words are unfamiliar
-                  to users, provide the meaning or translation so that users can understand them.
+               <h3>事例 7: 日本語で馴染みのない外来語</h3>
+               <p>日本語では、外来語に対してカタカナが用いられる。単語が利用者にとって馴染みがない場合、利用者が理解できるように意味又は翻訳を提供する。
                </p>
                <p>
                   <span lang="jp">アクセシビリティ（高齢者・障害者を含む全ての人が利用できること）は、Webサイトに不可欠である。</span>
                   
                </p>
-               <p>English translation: "Accessibility" (it can be accessed by all users including elderly
-                  people and people with disabilities) is an essential aspect of the Websites.
+               <p>アクセシビリティ (高齢者・障害者を含む全ての人が利用できること) は、ウェブサイトに不可欠である。
```

Comment:
英語の翻訳なので、原語のママとします。
```suggestion
               <p>English translation: "Accessibility" (it can be accessed by all users including elderly people and people with disabilities) is an essential aspect of the Websites.
```